pkg/test-infra/fixture/fixture.go

```diff
@@ -165,6 +166,9 @@ func init() {
 	flag.StringVar(&Context.LokiAddress, "loki-addr", "", "loki address. If empty then don't query logs from loki.")
 	flag.StringVar(&Context.LokiUsername, "loki-username", "", "loki username. Needed when basic auth is configured in loki")
 	flag.StringVar(&Context.LokiPassword, "loki-password", "", "loki password. Needed when basic auth is configured in loki")
+
+	flag.StringVar(&Context.ABTestConfig.Cluster1Version, "cluster1-version", "", "specify version for cluster1")
```

Comment:
```suggestion
	flag.StringVar(&Context.ABTestConfig.Cluster1Version, "abtest.a.image-name", "", "specify version for cluster1")
```

Reply:
I think "abtest.a.version" would be better, cause it only defines the cluster version but not the whole image name.

Reply:
OK for me